doc/repro.rst

```diff
@@ -11,9 +11,144 @@ Elsewhere:
+
+- One file, usually the main or index file **must** be included in the :code:`.. toctree::` directive in :file:`doc/index.rst`.
+
+- Extensive documentation for a project or model variant **should** be organized with headings, tables of contents, and if necessary split into several files.
+
+Ongoing projects
+----------------
+
+Documentation pages for ongoing projects **must** include a :code:`.. warning::` Sphinx directive at the top of the file indicating the code is under development.
+See e.g. :doc:`/transport/index`.
+This section **should** contain one or all of:
+
+- Link(s) to GitHub, including:
+
+  - A current tracking issue, which in turn can link to:
+
+    - Other issues and PRs where work occurs.
+    - Any of the items below.
+
+  - A project board, if any.
+  - A label for issues/PRs, if any.
+
+- Reference to all other locations where work is occurring, including any:
+
+  - Branch(es)—``main``, ``dev``, or any others—in :mod:`message-ix-models` or :mod:`message_data`.
+  - Fork(s) of these repos.
+  - Other repository/-ies separate :mod:`message-ix-models` or :mod:`message_data`
+
+  Not that this **does not** imply those should be made public, for instance prior to publication, if there are reasons not to; only that their existence and contents should be mentioned.
+
+This directive **must** be kept current, and removed once work is complete.
+
+Documentation for ongoing projects **should** be added to :mod:`message_ix_models`, even if some of the code or linked resources are in :mod:`message_data` or are otherwise private.
+
+Completed projects
+------------------
+
+Doc pages for completed projects **must** specify:
+
+- location(s) of scenarios, e.g.
+
+  - ixmp URLS giving the platform (‘database’), model name, scenario name, *and* version for any scenarios.
+    These **must** allow a reader to distinguish between ‘main’ or meaningful scenarios and other extras that should not be used.
+  - Specific external databases, Scenario Explorer instances, etc.
+
+- data sources,
+- reference to code used to prepare data,
+- any special parametrization or structure that is different from the RES or a referenced project, and
+- complete instructions to run all scenarios related to the project.
+
+Doc pages for completed projects **should** include a “Summary” section with all relevant items from the following list.
+This allows quick/at-a-glance understanding of the model configuration used for a completed project.
+These can be described *directly*, or by *reference*, for the latter, write “same as <other project>” and add a ReST link to a full description elsewhere.
+
+Example summary section
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Versions
+   See :ref:`versioning` for a complete discussion of the information to be recorded here.
+
+Regions
+   The regional aggregation used in the project.
+   Refer to one of the :doc:`message_ix_models:pkg-data/node`.
```

Comment:
This is rendered literally, i.e. `Refer to one of the message_ix_models:pkg-data/node.`, which does not seem to be the intention. Does the key `message_ix_models:pkg-data/node` exist?

Reply:
I believe this used to be an intersphinx cross-link from message_data to message-ix-models, but is now an internal link. So the `message_ix_models:` prefix is no longer needed. I should have removed it, and will do now.